account_invoice_report_due_list/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============================
+Account Invoice Report Due List
+===============================
+
+This module extends the functionality of account to show multiple due data
```

Comment:
This module extends the invoice report for adding information about the corresponding due dates and amounts.